clients/include/testing_gemm_strided_batched_ex.hpp

```diff
@@ -756,23 +723,24 @@ rocblas_status testing_gemm_strided_batched_ex_template(rocblas_operation transA
 
         for(rocblas_int i = 0; i < batch_count; i++)
         {
```

Comment:
No need for `.data()`. But both template arguments specified for `cblas_gemm<Td,Td>` so that all function arguments are cast correctly.